angular-todo/src/app/views/login/login.component.html

```diff
@@ -8,25 +8,27 @@
 of the MIT license. See the LICENSE file for details.
 -->
 
-<div class="cstm_container_v-centered container-fluid d-flex align-items-center">
-  <div class="w-100">
-    <app-back-home></app-back-home>
-    <div class="card shadow-sm p-5 mb-2 w-100">
-      <div class="cstm_form-icon align-self-center mb-3">
-        <ng-container *ngIf="!isWebAuthn; else fingerPrint">
-          <app-key-icon size="72px"></app-key-icon>
-        </ng-container>
+<ng-container *ngIf="!isCentralizedLogin">
```

Comment:
Sorry if I'm just missing this, but where's the `else` for this condition? If it's not centralized login, what renders?

Reply:
hi justin, if it is centralized login it redirects to ForgeRock AM, or another web application to handle authentication. If not centralized renders the form to fill out with username and password